templates/node/node--event--card.html.twig

```diff
@@ -24,6 +24,12 @@
   reference_card__image: 'true',
 } %}
   {% block reference_card__image %}
-    {{ content.field_teaser_media }}
+
+    {% if content.field_teaser_media[0] %}
+      {{ content.field_teaser_media }}
+    {% else %}
+      {{ drupal_entity('media', getCoreSetting('image_fallback.teaser'), 'card_secondary_3_2') }}
```

Comment:
Our plan is to give sites a fallback image out of the box. But we are also giving the site admins the ability to change this. So it is possible that `getCoreSetting('image_fallback.teaser')` will return NULL and this will throw a fatal error.

I suggest we change these `else` to `elseif` with a condition. 
Or alternatively, we can create a custom twig function that does all of the business logic for us (but this could be overkill).

Reply:
Fixed, good catch! Thanks!